### PR TITLE
[es]: Update php and php-inline.json

### PR DESCRIPTION
**Pull request by luisprmat**

### CODE REVIEW

File: locales/es/_excludes.json [hunk 1]
```diff
@@ -48,6 +59,7 @@
     "Guernsey",
     "Guinea",
     "Guyana",
+    "Hawai",
```
Comment (andrey-helldar) [inline locales/es/_excludes.json:62]:
```suggestion
    "Hawaii",
```

File: locales/es/_excludes.json [hunk 2]
```diff
@@ -48,6 +59,7 @@
     "Guernsey",
     "Guinea",
     "Guyana",
+    "Hawaii",
```
Comment (andrey-helldar) [inline locales/es/_excludes.json:62]:
```suggestion
```

**Comment by andrey-helldar** [comment]
What is the correct word for "Hawaii" in Spanish?

https://github.com/luisprmat/lang/blob/bc5ea57bc63510257e90b972b2baf0228955563d/locales/es/json.json#L355

**Comment by luisprmat** [comment]
Sorry, Hawaii in spanish is Hawái

**Comment by andrey-helldar** [comment]
In that case, I hope you don't mind that I changed the value in the file 🙂

**Review by andrey-helldar — APPROVED** [review]
Muchas gracias

**Comment by luisprmat** [comment]
> In that case, I hope you don't mind that I changed the value in the file 🙂

No problem, I'll continue on the missing translations soon

**Pull request merged**